curvefs/conf/tools.conf

```diff
@@ -35,6 +35,7 @@ s3.bucket_name=bucket
 s3.blocksize=4194304
 s3.chunksize=67108864
 s3.useVirtualAddressing=false
+s3.objectPrefix=0
```

Comment:
With this design, users can only use the metadata format of the specified version, and cannot upgrade it?

Reply:
Yes, users cann't change the prefix once the prefix is decided just as the s3.blocksize. If the users want to change the prefix, There are two methods, one is to create another fs, and sync form the previous fs. The other is to sync the s3 object from one bucket to another bucket, While syncing the object, change the object format. Finally,  using the tools to change the bucket and prefix.